lib/parse/audio/audioData.ts

```diff
@@ -83,13 +84,13 @@ export const parseAudioData = (
         parent.node.date)) ||
     audioBroadcastDate ||
     dataDate;
-  const date = broadcastDate && new Date(`${broadcastDate}T00:00:00`);
+  const date = broadcastDate && sanitizeIso8601Date(broadcastDate);
```

Comment:
this definitely feels better, a better abstraction for date handling

Reply:
Looks like you're sanitizing this date, then creating a Date out of it on line 93 below.
I was expecting both calling sanitize and then creating a Date out of the result to happen in the `DateTime` object, is there a reason to do that here without a `DateTime`?